tests/test_memoryfs.py

```diff
@@ -14,6 +16,9 @@
     tracemalloc = None
 
 
+@pytest.mark.skipif(
+    not tracemalloc, reason="`tracemalloc` isn't supported on this Python version."
```

Comment:
Does moving this up to here now mean the whole of `TestMemoryFS` doesn't get tested if `tracemalloc` isn't available, whereas previously it'd only skip `test_close_mem_free`?

Reply:
Oooh yeah, good point